contracts/minter/src/contract.rs

```diff
@@ -36,7 +36,9 @@ const INSTANTIATE_SG721_REPLY_ID: u64 = 1;
 const MAX_TOKEN_LIMIT: u32 = 10000;
 const MAX_PER_ADDRESS_LIMIT: u32 = 50;
 const MIN_MINT_PRICE: u128 = 50_000_000;
+const AIRDROP_MINT_PRICE: u128 = 15_000_000;
 const MINT_FEE_PERCENT: u32 = 10;
+const AIRDROP_MINT_FEE_PERCENT: u32 = 100;
```

Comment:
was a bit confusing to me since its the Network Fee percent maybe just adding some comments to these consts would be nice

Reply:
added helpful comment